python/nav/web/api/v1/auth.py

```diff
@@ -58,6 +62,56 @@ def authenticate(self, request):
             return request.account, None
 
 
+class JWTToken(dict):
+    pass
+
+
+class JWTAuthentication(JSONWebTokenAuthentication):
```

Comment:
Add a docstring to explain that this class enforces the JWT token requirements of the NAV API

Reply:
Name it so also: NAVJWTAuthentication.

Reply:
got rid of this problem :)